spec/unit/recipes/default_spec.rb

```diff
@@ -3,8 +3,8 @@
 describe 'solr-jetty::default' do
 
   platforms = {
-    'centos' => ['6.6', '7.0'],
-    'ubuntu' => ['14.04']
+    'centos' => ['6.8', '7.0'],
+    'ubuntu' => ['14.04', '16.04']
```

Comment:
Should these old versions still be here?  I thought we're only supporting 7.0 and 16.04, now.